devito/types/relational.py

```diff
@@ -5,7 +5,25 @@
 __all__ = ['Le', 'Lt', 'Ge', 'Gt', 'Ne']
 
 
-class Le(sympy.Le):
+class AbstractRel(object):
+    """
+    Abstract mixin class for objects subclassing sympy Relationals.
```

Comment:
SymPy

Reply:
or `sympy.Relational` (better?)

Reply:
better yes, this was my original thought actually, so glad you agree.